README.md

```diff
@@ -0,0 +1,172 @@
+# build-ui-cl
+
+## BuildUI
+
+BuildUI is a component Library built using HTML and CSS.You can use Component Library in your projects With the help of Component Library ,you don't need to code from scratch,You just pick component from library Which ever you need and start using it .
+
+## Usage /Installation ##
+
+First,you need to import the links in this page :
+https://build-ui-component-library.netlify.app/index.html
+
+Import here links in your file to use Component Library
+
+## How to run app ##
+
+To use any Component,first you need to import the above link in your file and then just copy and paste the HTML code of the components.
+
+## video ##
+
+
+https://user-images.githubusercontent.com/87228863/155079451-288f1cb5-bd5a-4f43-9688-125eb13bef61.mp4
+
+Here video will come
+
+## List of Components ##
+
+1. Alert
+2. Avatar
+3. Badge
+4. Button
+5. Card
+6. Image
+7. Input
+8. Text-utilites
+9. Lists
+10. Navigation-Desktop
+11. Modal
+12. Rating
+13. Toast/Snackbar
+14. Simplified Grid
+
+
+## 1. Alerts ##
+
+Alert Provide contextual feedback messages for typical user actions with the handful of available and flexible alert messages.
+
+We have 6 types of alerts
+
+• Primary Alert
+• Secondary Alert
+• White Alert
+• Danger Alert
+• Warning Alert
+• Success Alert
+
+## 2. Avatar ##
+
+Avatars are images that users can set as their profile picture.
+
+We have 4 types of avatar
+
+• Round Avatar
+• Square Avatar
+• Round Text Avatar
+• Square Text Avatar
+
+
+## 3. Badge ##
+
+Badges provide additional information about a component like it's state whether it is active or not.
+
+we have 2 types of Badge
+
+• Badges on icons
+• Badges on Avatar
+
+## 4. Button ##
+
+Buttons are main component of any website.They are highly used component.
+
+We have 5 buttons
+
+• Primary Buttons
+• Link Buttons
+• Icon Buttons
+• Floating Buttons
```

Comment:
typo probably, you have mentioned 5 buttons

Reply:
fixed